web/components/SearchBarLocation.vue

```diff
@@ -1,28 +1,29 @@
 <template>
   <v-autocomplete
     v-model="keystring"
+    append-icon="mdi-magnify"
```

Comment:
didn't realize that this causes the icon to flip around.  can we get it to stop doing that, or move this icon somewhere that it won't have this behaviour?

Reply:
found the appropriate selector and added it here :)